src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationViewTaggerProvider.cs

```diff
@@ -92,14 +91,20 @@ protected override Task ProduceTagsAsync(TaggerContext<IClassificationTag> conte
             Debug.Assert(context.SpansToTag.IsSingle());
 
             var spanToTag = context.SpansToTag.Single();
-            var document = spanToTag.Document;
 
-            if (document == null)
+            // Attempt to get a classification service which will actually produce the results.
+            // If we can't (because we have no Document, or because the language doesn't support
+            // this service), then bail out immediately.
+            if (_classificationService == null)
```

Comment:
If we don't have a service, should we even have created a tagger in the first place? If the CreateTagger returns null to the editor, we won't even be participating.